File: 268-missing-number/missing-number.c
int missingNumber(int* nums, int numsSize) {
      
        int expectation = numsSize* (numsSize+1)/2;
        int actualsize = 0;
        for(int i=0;i<numsSize;i++){
            actualsize += nums[i];
        }
        return expectation-actualsize;
}

File: 268-missing-number/missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    var n = nums.length
    var expectation = n* (n+1)/2;
    var actualsize = 0;
        for(var i=0;i<n;i++){
            actualsize += nums[i];
        }
        return expectation-actualsize;
};